system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_flow_test.clj

```diff
@@ -101,7 +101,8 @@
         json-body (json/generate-string bulk-update-body)
         {:keys [task-id]} (ingest/bulk-update-collections "PROV1" bulk-update-body)
         _ (qb-side-api/wait-for-terminal-states)
-        status-response (ingest/bulk-update-task-status "PROV1" task-id)]
+        status-response (ingest/bulk-update-task-status "PROV1" task-id)
+        status-response (dissoc status-response :created-at)]
```

Comment:
Sorry running out of time to do sophisticated time comparisons since it involves time synchronization.   Added demo to demonstrate that created-at contains the timestamp when bulk update happens. Chris said maybe someone else could add this part next week.